packages/vscode-apollo/src/statusBar.ts

```diff
@@ -1,36 +1,37 @@
-import {
-  window,
-  StatusBarItem,
-  StatusBarAlignment,
-  ExtensionContext,
-  commands
-} from "vscode";
-import { LanguageClient } from "vscode-languageclient";
+import { window, StatusBarAlignment } from "vscode";
 
 export default class ApolloStatusBar {
-  private statusBarItem: StatusBarItem = window.createStatusBarItem(
-    StatusBarAlignment.Right
-  );
+  private _statusBarItem = window.createStatusBarItem(StatusBarAlignment.Right);
```

Comment:
Why did you go with this convention here? Just for consistency across the project or what? Do you think it's necessary given we're already declaring it private with TS?

Reply:
The convention is useful when you're using private variables with a getter (See `get statusBarItem` below). The value of keeping this private is debatable, but it does protect the internal instance of the `statusBarItem` from being set externally.

Reply:
Resolved with 2afa864d200f34299fe9077d2313f3ba7814f031